.github/workflows/build.yaml

```diff
@@ -9,10 +9,11 @@ on:
   pull_request:
     paths: 
       - Dockerfile
+      - scripts/**
 
 jobs:
     build:
-      runs-on: ubuntu-20.04
+      runs-on: macos-11
```

Comment:
Not sure we want to run this on Mac. What's the reason behind this?

Reply:
Because linux is out of space when running test. But looks macOS not having docker installed.

Reply:
```
Run df -h
Filesystem      Size  Used Avail Use% Mounted on
/dev/root        84G   53G   32G  [6](https://github.com/react-native-community/docker-android/runs/6121950579?check_suite_focus=true#step:3:6)3% /
devtmpfs        3.4G     0  3.4G   0% /dev
tmpfs           3.4G  4.0K  3.4G   1% /dev/shm
tmpfs           695M  1.1M  694M   1% /run
tmpfs           5.0M     0  5.0M   0% /run/lock
tmpfs           3.4G     0  3.4G   0% /sys/fs/cgroup
/dev/loop0       62M   62M     0 100% /snap/core20/1405
/dev/loop1       68M   68M     0 100% /snap/lxd/22[7](https://github.com/react-native-community/docker-android/runs/6121950579?check_suite_focus=true#step:3:7)53
/dev/loop2       44M   44M     0 100% /snap/snapd/15177
/dev/sda15      105M  5.2M  100M   5% /boot/efi
/dev/sdb1        14G  4.1G  [9](https://github.com/react-native-community/docker-android/runs/6121950579?check_suite_focus=true#step:3:9).0G  32% /mnt
```